storage/txcache/txMeasures.go

```diff
@@ -0,0 +1,27 @@
+package txcache
+
+import "github.com/ElrondNetwork/elrond-go/data"
+
+const estimatedSizeOfBoundedTxFields = uint32(128)
```

Comment:
why not making this const directly int64?

Reply:
Done.